test.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
-var assert = require('assert');
-var arrayDifference = require('./');
+import test from 'ava';
+import arrayDifference from './';
```

Comment:
the `var` name has not been modified, not sure about the improvement of the suggested change. I would defer it to @sindresorhus